doc/build-macos.md

```diff
@@ -59,10 +59,15 @@ Build Gridcoin
 
         make check
 
-4.  You can also create a .dmg that contains the .app bundle (optional):
+4. To have terminal give full readout if desired: 
 
-        make deploy
+        make V=1 -j #number_of_cores_whatever >& build.log
```

Comment:
No need to make again, you run the make command on line 49 in this guide. Add the line 

    To have terminal give full readout if desired:

On line 50.